drivers/sensor/lis2dh/lis2dh.h

```diff
@@ -200,6 +205,7 @@ struct lis2dh_config {
 #endif /* CONFIG_LIS2DH_TRIGGER */
 	bool is_lsm303agr_dev;
 	bool disc_pull_up;
+	bool anym_on_int1;
```

Comment:
At some point I think this should get packed into a bit field to avoid bloating the struct, but I didn't count the bytes to see if it gets bigger in this PR.